stable/rabbitmq/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "rabbitmq.fullname" . }}
+  {{- if .Values.metrics.serviceMonitor.namespace }}
+  namespace: {{ .Values.metrics.serviceMonitor.namespace }}
+  {{- end }}
+  labels:
+    app: {{ template "rabbitmq.name" . }}
+    chart: {{ template "rabbitmq.chart" .  }}
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+    {{- if .Values.metrics.serviceMonitor.additionalLabels }}
+{{ toYaml .Values.metrics.serviceMonitor.additionalLabels | indent 4 }}
+    {{- end }}
+spec:
+  endpoints:
+  - port: metrics
+    interval: 30s
+    {{- if .Values.metrics.serviceMonitor.honorLabels }}
+    honorLabels: true
+    {{- end }}
+    {{- if .Values.metrics.serviceMonitor.relabellings }}
+    metricRelabelings:
+{{ toYaml .Values.metrics.serviceMonitor.relabellings | indent 6 }}
+    {{- end }}
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
```

Comment:
Shouldn't you use `{{ .Values.metrics.serviceMonitor.namespace }}` here?

Reply:
No, is correct right that. That parameter must match the namespace of [rabbimq service resource](https://github.com/helm/charts/blob/master/stable/rabbitmq/templates/svc.yaml).

See these other charts for check:
* https://github.com/helm/charts/blob/master/stable/postgresql/templates/servicemonitor.yaml#L28
* https://github.com/helm/charts/blob/master/stable/mongodb/templates/prometheus-service-monitor.yaml#L25
* https://github.com/helm/charts/blob/master/stable/redis/templates/metrics-prometheus.yaml#L29